go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/terraform-plugin-docs v0.13.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.19.0
-	github.com/nobl9/nobl9-go v0.4.0
+	github.com/nobl9/nobl9-go v0.6.0
```

Comment:
just an idea for a future task - we should add dependabot / renovate to this repo